core/src/components/radio/test/legacy/a11y/radio.e2e.ts

```diff
@@ -6,10 +6,11 @@ import { configs, test } from '@utils/test/playwright';
  */
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
   test.describe(title('radio: a11y'), () => {
-    // TODO(FW-5218)
-    test.fixme('tabbing should switch between radio groups', async ({ page, pageUtils }) => {
+    test.beforeEach(async ({ page, skip }) => {
+      skip.browser('webkit', 'Tabbing is flaky in Safari');
```

Comment:
You might need this fix? https://github.com/ionic-team/ionic-framework/blob/331c08aad542de158e53ed351705d4c396bb4e90/core/src/components/accordion/test/a11y/accordion.e2e.ts#L13

Since you were unable to replicate the flakiness, I would try adding the fix and re-enabling the test on all browsers. If it flakes again, we can always add the webkit skip back in.

Reply:
The `pressKeys` [function](https://github.com/ionic-team/ionic-framework/blob/331c08aad542de158e53ed351705d4c396bb4e90/core/src/components/radio/test/legacy/a11y/radio.e2e.ts#L16) has been customized to update the Tab if it's Webkit. So we [don't need to check](https://github.com/ionic-team/ionic-framework/blob/331c08aad542de158e53ed351705d4c396bb4e90/core/src/utils/test/press-keys.ts#L106) for browsers anymore.

Reply:
Ah, forgot about that! Normally I'd still ask that we remove the webkit skip, but since this is a legacy test, it doesn't matter that much IMO.